### ISSUE TITLE
Home Assistant Device class for Binary Sensors

### ISSUE THREAD

**Issue by SNoof85** [fmartinou/whats-up-docker#100]
Hi,

Just be running WUD, a very cool piece of stuff btw, and integrated it to HA. It would be great to use now the `device_class: update` for Binary sensors. This class is designed for updates available or not wich is the main goal of all binary sensors created with WUD !

See : https://www.home-assistant.io/integrations/binary_sensor/#device-class
And can be pushed with MQTT Discovery : 'dev_cla' or 'device_class' set to `update`

**Comment by fmartinou** [comment]
Hi,

Thanks for pointing me to this feature; I'll definitely add it!

**Comment by fmartinou** [comment]
It's now implemented and it will be part of the next release.

In the mean time, you can give it a try with version `fmartinou/whats-up-docker:feature__100`.

(please also notice that you may have to remove the existing wud entities from home-assistant first to see the new device_class properly applied)

<img width="210" alt="image" src="https://user-images.githubusercontent.com/16061231/151038040-2e2bf4ff-737f-46d3-83cd-106071596b5c.png">

**Comment by SNoof85** [comment]
Hey tried it and looks not so good :
```
09:23:54.906  INFO whats-up-docker: What's up Docker? is starting (version = feature__100)                                                                                                                         
09:23:54.909  INFO whats-up-docker/store: Load store from (/store/wud.json)                                                                                                                                        
09:23:54.954  INFO whats-up-docker/store: Migrate data from version latest to version feature__100                                                                                                                 
09:23:54.958  INFO whats-up-docker/prometheus: Init Prometheus module                                                                                                                                              
09:23:55.118  INFO whats-up-docker/registry: Register all components of kind trigger for provider mqtt                                                                                                             
09:23:55.398  INFO whats-up-docker/trigger.mqtt.mosquitto: Register trigger with configuration {"url":"mqtt://192.168.1.3:1883","user":"REDACTED","password":"REDACTED","hass":{"enabled":true,"prefix":"homeassistant"
},"topic":"wud/container","threshold":"all","mode":"single","once":true,"simpletitle":"New ${kind} found for container ${name}","simplebody":"Container ${name} running with ${kind} ${local} can be updated to ${k
ind} ${remote}\n${link}","batchtitle":"${count} updates available","key":"p******t"}                                                                                                                               
09:23:56.522  INFO whats-up-docker/registry.hub: Register registry with configuration {}                                                                                                                           
09:23:56.531  INFO whats-up-docker/registry.ghcr: Register registry with configuration {}                                                                                                                          
09:23:58.724  INFO whats-up-docker/watcher.docker.local: Register watcher with configuration {"cron":"0 */6 * * *","socket":"/var/run/docker.sock","watchall":true,"port":2375,"watchbydefault":true,"watchevents":
true}                                                                                                                                                                                                              
09:23:58.739  INFO whats-up-docker/watcher.docker.local: Cron scheduled (0 */6 * * *)                                                                                                                              
09:23:58.791  INFO whats-up-docker/registry: No authentication configured => Allow anonymous access                                                                                                                
09:23:58.802  INFO whats-up-docker/authentication.anonymous.anonymous: Register authentication with configuration {}                                                                                               
09:23:58.826  WARN whats-up-docker: Anonymous authentication is enabled; please make sure that the app is not exposed to unsecure networks                                                                         
09:23:58.858  INFO whats-up-docker/api: API/UI exposed on port 3000                                                                                                                                                
09:23:59.770  INFO whats-up-docker/watcher.docker.local: Cron started (0 */6 * * *)                                                                                                                                
09:23:59.798  INFO whats-up-docker/watcher.docker.local: Listening to docker events                                                                                                                                
09:23:59.902  WARN whats-up-docker/watcher.docker.local: Cannot get a reliable tag for this image [sha256:db209923532eb78b1ea206277ca8f50498c526709ac572b28cee7a8f8f5d4465]                                        
09:23:59.917  INFO whats-up-docker/trigger.mqtt.mosquitto: Remove hass device [id=wud_container_local_wud]                                                                                                         
09:24:01.184  INFO whats-up-docker/trigger.mqtt.mosquitto: Add hass device [id=wud_container_local_wud]                                                                                                            
09:24:16.434  INFO whats-up-docker/watcher.docker.local: Cron finished (12 containers watched, 0 errors, 1 available updates)
```

And in HASS side : 
![image](https://user-images.githubusercontent.com/25659602/151128844-65b728d5-d7b8-4c99-95fe-1f1e932fe1c3.png)


So, WUD says I have an update, and HASS says it's up-to-date

**Comment by fmartinou** [comment]
Euh...

Just to be sure, may you first:
1. Remove all mqtt topics related to WUD (`homeassistant/binary_sensor/wud_*` and `wud/*`)
2. Remove all wud entities from HA
3. Remove the wud `/store/wud.json` file
4. Restart wud

And see whether it still weird?

Cause with a fresh install, I can confirm it's working 🤔 .

**Comment by SNoof85** [comment]
I agree, that weird...

I guess I need to clean a bit averything like you said. I'm in the office today and have not all the tools to check it now. Will try it tonight and will report. 

Having a look at the code and I'm pretty sure it's coming from my setup :)

**Comment by fmartinou** [comment]
All right, keep me informed 😃 .

**Comment by fmartinou** [comment]
Available in version `5.14.1`.

**Comment by SNoof85** [comment]
After a good cleanup, works as expected !
Merci !
